Sources/TuistSupport/Logging/Logger.swift

```diff
@@ -47,35 +63,51 @@ public enum LogOutput {
             handler = StandardLogHandler.self
         }
 
-        if config.verbose {
+        switch config.loggerLevel {
+        case .verbose:
             LoggingSystem.bootstrap(handler.verbose)
-        } else {
+        case .silent:
+            LoggingSystem.bootstrap(handler.silent)
+        case .default:
             LoggingSystem.bootstrap(handler.init)
         }
     }
 }
 
-// A `VerboseLogHandler` allows for a LogHandler to be initialised with the
-// `debug` logLevel.
-protocol VerboseLogHandler: LogHandler {
+// A `VariableLevelLogHandler` allows for a LogHandler to be initialised with the
+// `debug` or `error` logLevel.
+protocol VariableLevelLogHandler: LogHandler {
```

Comment:
I'm open to any alternative naming proposals

Reply:
How about `LogLevelHandler`?

Reply:
Wouldn't it suggest that it's a type that "handles log levels" instead of a type that "handles logs"?
Maybe `ConfigurableLevelLogHandler` / `LevelInitializableLogHandler`?